packages/components/src/slot-fill/test/slot.js

```diff
@@ -34,20 +34,20 @@ class Filler extends Component {
 
 describe( 'Slot', () => {
 	it( 'should render empty Fills', () => {
-		const element = mount(
+		const tree = ReactTestRenderer.create(
```

Comment:
All those tests are much harder to read since snapshots were introduced. Is it possible to rewrite them to use inline check against generated HTML?

Reply:
Let's discuss it in https://github.com/WordPress/gutenberg/pull/11314